README.md

```diff
@@ -45,47 +46,41 @@ pip install -i https://test.pypi.org/simple/ kymatio==0.0.1
 
 
 
-The software was tested on Linux with anaconda Python 3 and
+The software was tested on Linux with Anaconda Python 3 and
 various GPUs, including Titan X, 1080s, 980s, K20s, and Titan X Pascal.
 
 The software uses PyTorch + NumPy FFT on CPU, and PyTorch + CuPy + CuFFT on GPU.
 
 
-If using this code for your research please cite our paper:
+If you use this code in your work please cite our paper:
 
-The scattering authors, [*kymatio: Fast Scattering in 1-D,2-D,3-D*]()
+The scattering authors, [*Kymatio: Fast Scattering in 1-D,2-D,3-D*]()
```

Comment:
Or remove "The scattering authors"?

Reply:
this will go away in the next iteration of the README anyway
(which will include a list of resources)
no need to do everything at the same time

Reply:
Ok. Will leave this, for now.